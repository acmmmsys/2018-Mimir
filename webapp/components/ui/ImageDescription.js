import React, { Component } from 'react';

import 'style/ui/ImageDescription.scss';

class ImageDescription extends Component {
    render() {
        return (
            <div className={'image-description'}>
                { this.props.description }            
            </div>
        );
    }
}

export default ImageDescription;