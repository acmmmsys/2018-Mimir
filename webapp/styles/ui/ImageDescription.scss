
.image-description {
    width: 100%;
    text-align: center;
}