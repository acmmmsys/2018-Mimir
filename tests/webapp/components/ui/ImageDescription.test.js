import React from 'react';
import ImageDescription from '../../../../webapp/components/ui/ImageDescription'
import renderer from 'react-test-renderer';
import configureStore from 'redux-mock-store'
 
const mockStore = configureStore();

let store;
let wrapper;

test('ImageDescription does render', () => {
  const wrapper = shallow(
      <ImageDescription store={mockStore(initialState)}>This is a test!</ImageDescription>
  );
  expect(wrapper).toMatchSnapshot();
});
