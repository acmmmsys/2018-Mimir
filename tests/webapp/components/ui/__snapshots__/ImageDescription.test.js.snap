// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImageDescription does render 1`] = `
<div
  className="image-description"
/>
`;
